content/post/kubedb-v2024.06.04/index.md

```diff
@@ -0,0 +1,788 @@
+
+#### Restart
+Restart ops request is used to perform a smart restart of the Druid cluster. An example YAML is provided below:
+```yaml
+apiVersion: ops.kubedb.com/v1alpha1
+kind: DruidOpsRequest
+metadata:
+  name: druid-restart
+  namespace: demo
+spec:
+  type: Restart
+  databaseRef:
+    name: druid
+```
+
+#### Vertical Scaling:
+Vertical Scaling allows you to vertically scale the Druid nodes (ie. pods). The necessary information required for vertical scaling, must be provided in the `spec.verticalScaling` field.
+An example yaml is provided below:
+```yaml
+apiVersion: ops.kubedb.com/v1alpha1
+kind: DruidOpsRequest
+metadata:
+  name:  dops-vscale
+  namespace: demo
+spec:
+  type: VerticalScaling
+  databaseRef:
+    name: druid
+  verticalScaling:
+    middleManagers:
+      resources:
+        limits:
+          memory: 2560Mi
+        requests:
+          cpu: 900m
+          memory: 2560Mi
+    coordinators:
+      resources:
+        limits:
+          memory: 1Gi
+        requests:
+          cpu: 600m
+          memory: 1Gi
+```
+
+#### Volume Expansion:
+Volume Expansion is used to expand the storage of the Druid nodes (ie. pods). The necessary information required for volume expansion, must be provided in `spec.volumeExpansion` field. Example YAML:
+
+```yaml
+apiVersion: ops.kubedb.com/v1alpha1
+kind: DruidOpsRequest
+metadata:
+  name: dops-vol-exp
+  namespace: demo
+spec:
+  type: VolumeExpansion
+  databaseRef:
+    name: druid
+  volumeExpansion:
+    mode: "Online"
+    middleManagers: 4Gi
+    historicals: 4Gi
+```
+
+### Autoscaler
+Support for Druid Compute Autoscaling for all druid nodes (i.e. pods) and Storage Autoscaling for druid data nodes (i.e. `historicals` & `middleManagers` pod) has also been added. To enable autoscaling with a particular specification users need to install a Custom Resource Object of Kind `DruidAutoscaler`. DruidAutoscaler is a Kubernetes Custom Resource Definitions (CRD). It provides a declarative configuration for autoscaling Druid compute resources and storage of database components in a Kubernetes native way.  Some sample DruidAutoscaler CRs for autoscaling different components of database is given below:
+```yaml
+apiVersion: autoscaling.kubedb.com/v1alpha1
+kind: DruidAutoscaler
+metadata:
+  name: druid-as
+  namespace: demo
+spec:
+  databaseRef:
+    name: druid  
+  compute:
+   middleManagers:
+      trigger: "On"
+      podLifeTimeThreshold: 1m
+      minAllowed:
+        cpu: 600m
+        memory: 3Gi
+      maxAllowed:
+        cpu: 1
+        memory: 5Gi
+      resourceDiffPercentage: 20
+      controlledResources: [ "cpu", "memory" ]
+   historicals:
+      expansionMode: "Online"
+      trigger: "On"
+      usageThreshold: 2
+      scalingThreshold: 50
+```
+## Elasticsearch
+**New Version**: We have added two new versions. One for Elasticsearch and the other for Opensearch. Newly available versions are `xpack-8.13.4`(Elasticsearch) and `opensearch-2.14.0`(Opensearch)
```

Comment:
**New Version** -> **New Version support**
This is a general change for all dbs

Reply:
Also do not give any description.  
New Version support: `xpack-8.13.4`(Elasticsearch), `opensearch-2.14.0`(Opensearch)

is more than enough